etc/adminhtml/system.xml

```diff
@@ -94,6 +94,21 @@
                         <config_path>payment/omise_offsite_alipay/title</config_path>
                     </field>
                 </group>
+                <group id="omise_offsite_citipoints" translate="label" sortOrder="160" showInDefault="1" showInWebsite="1" showInStore="1">
+                    <label>Enable Citi Points Solution</label>
```

Comment:
Maybe Citi Points solution is a bit ambiguous? As it is not our marketing term for this feature. 
Maybe "Pay with City Points Solution"?

Reply:
Agree that is way better name

Reply:
Removing 'Solution' is probably better

Reply:
done in #232

Reply:
Apparently "Citi points" is not correct - according to Robin, the correct name is "City Pay with Points"